content/en/docs/contributing/release-process.md

```diff
@@ -10,6 +10,31 @@ new release of cert-manager.
 
 ## Prerequisites
 
+<!--
+About "alert" and "pageinfo": https://www.docsy.dev/docs/adding-content/shortcodes/#shortcode-helpers
+-->
+
+{{% pageinfo color="warning" %}}
+
+⛔️ Do not proceed with the release process if you do not satify to one of the following:
+
+1. You need to be a GitHub "owner" on the cert-manager project (if you see the
+   "Settings" tab when looking at the cert-manager project, you **are** an
+   owner). To become a GitHub "owner" of the cert-manager project, [open a
+   PR](https://github.com/jetstack/platform-board/pulls/new) with a link to
+   here.
+2. You need to be added as an "Owner" to the GCP project
+   [cert-manager-release](https://console.cloud.google.com/?project=cert-manager-release).
+   To check if you do have access, try opening [the Cloud Build
+   page](https://console.cloud.google.com/cloud-build?project=cert-manager-release).
+   To get "Editor" access to the GCP project, you can copy-paste [this PR
+   example](https://github.com/jetstack/platform-board/issues/373) into a new PR
```

Comment:
I think this is not publicly accessible?

Reply:
Oops, good point 😞

I added a warning about being at jetstack in order to ask for the permissions; does it make sense?

I would (really) prefer not to have this "must be at jetstack" thing. In my mind, we should be able to ask for this access as long as you are listed as a code owner of cert-manager. In this case, I think you should be able to request access by creating a PR on https://github.com/jetstack/cert-manager and have it reviewed by Zee (IT team at Jetstack).

What do you think?

Reply:
If that's OK for you, I'll proceed with merging this PR

Feel free to let me know here if you feel we should rephrase this or dig deeper into how to make the process friendly to people external to jetstack (I'll raise this issue during tonight's biweekly dev meeting)